pkg/controllers/provisioning/binpacking/packer.go

```diff
@@ -93,17 +93,21 @@ func (p *Packer) Pack(ctx context.Context, schedule *scheduling.Schedule, instan
 			continue
 		}
 		key, err := hashstructure.Hash(packing, hashstructure.FormatV2, &hashstructure.HashOptions{SlicesAsSets: true})
-		if err == nil {
+		if err != nil {
+			logging.FromContext(ctx).Errorf("Hashing packings to determine node batches")
+			packings = append(packings, packing)
+		} else {
```

Comment:
I'm confused here. Don't we want to hard abort if err!= nil?

Reply:
Not necessarily. If a provisioner changed in such a way that caused an error in the hashing lib, we could recover and just not batch the packings.

Reply:
I can't really think of why hashstructure.Hash would actually return an error ... it seems like hard abort is safer